cves/kernel/CVE-2018-20169.yml

```diff
@@ -339,10 +266,53 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: false
+  stacktrace_with_fix: false
+description: The USB subsystem fails to check size of extra descriptor by the
+  minimum and maximum size allowed, to prevent from invalid data being sent by a
+  USB device and crashing the host device.
```

Comment:
I think it could be explained in more detail and lesser technical terms? Like explaining what the extra descriptor is.

Reply:
Reworded to be slightly clearer